sendmail.py

```diff
@@ -2,19 +2,33 @@
 from email.message import EmailMessage
 from decouple import config
 
+
 def send(subject, recipient, body):
+    # was older gmail variables set?
     gmail_user = config('GMAIL_USER')
     gmail_password = config('GMAIL_PWD')
 
+    email_server = config('EMAIL_SERVER')
+    email_server_port = config('EMAIL_SERVER_PORT')
+    email_user = config('EMAIL_USER')
+    email_password = config('EMAIL_PASSWORD')
+
+    if gmail_user:
```

Comment:
Since the gmail_user config is no longer in use, I would recommend using a different condition. Can either check email_user for 'gmail.com' or perhaps simply if !email_server. 

```
if !email_server:
  email_server = 'smtp.gmail.com'
  email_server_port = 465
```